Python/Python.sublime-syntax

```diff
@@ -228,63 +228,63 @@ contexts:
         2: keyword.control.flow.with.python
       push: with-body
     # except ... as ...:
-    - match: \b(except)\b
-      scope: keyword.control.flow.except.python
+    - match: \bexcept\b
+      scope: keyword.control.exception.except.python
```

Comment:
`keyword.control.exception.catch` is the general term I proposed (and `catch` is used more commonly).

Should be adjusted for the other occurances as well.